### ISSUE TITLE
Jetty 10 documentation generating .png files within src directory.

### ISSUE THREAD

**Issue by lachlan-roberts** [eclipse/jetty.project#5805]
**Jetty version**
10.0.x

**Description**

In the operations-guide directory I'm getting a lot of generated .png .svg files. These files should probably be generated in the target directory so they're ignored by git.

```
Lachlan@Lachlans-MBP: ~/webtide/src/jetty-10.0.x/jetty-documentation/src/main/asciidoc/operations-guide (jetty-10.0.x-5229-WebSocketDocumentation)^1
[513]: ls -la
total 272
drwxr-xr-x  35 Lachlan  staff   1120 Dec  3 17:04 .
drwxr-xr-x   8 Lachlan  staff    256 Oct  6 11:02 ..
drwxr-xr-x   3 Lachlan  staff     96 Dec  3 17:02 .asciidoctor
-rw-r--r--   1 Lachlan  staff    210 Sep 24 08:47 .asciidoctorconfig
drwxr-xr-x   3 Lachlan  staff     96 Nov 11 09:26 annotations
drwxr-xr-x   8 Lachlan  staff    256 Dec  3 10:34 begin
drwxr-xr-x   3 Lachlan  staff     96 Sep 24 08:47 contexts
drwxr-xr-x  10 Lachlan  staff    320 Nov 11 09:26 deploy
-rw-r--r--   1 Lachlan  staff   8179 Dec  3 17:04 diag-2830e38460d606111888c5df2a645163.png
-rw-r--r--   1 Lachlan  staff   3293 Dec  3 17:04 diag-2830e38460d606111888c5df2a645163.svg
-rw-r--r--   1 Lachlan  staff   8136 Dec  3 17:04 diag-70817cc5c7137abc9759cdcd3edbb658.png
-rw-r--r--   1 Lachlan  staff   3021 Dec  3 17:04 diag-70817cc5c7137abc9759cdcd3edbb658.svg
-rw-r--r--   1 Lachlan  staff   8090 Dec  3 17:04 diag-7387dfbb6f9fb9cfa951a697aca488a8.png
-rw-r--r--   1 Lachlan  staff   2919 Dec  3 17:04 diag-7387dfbb6f9fb9cfa951a697aca488a8.svg
-rw-r--r--   1 Lachlan  staff  20939 Dec  3 17:04 diag-8a7fb4bb9c84453a4bf01a24558e8026.png
-rw-r--r--   1 Lachlan  staff   7211 Dec  3 17:04 diag-8a7fb4bb9c84453a4bf01a24558e8026.svg
-rw-r--r--   1 Lachlan  staff  17220 Dec  3 17:04 diag-9f323b812fd1edc8217715ca0c2ba8e9.png
-rw-r--r--   1 Lachlan  staff   2975 Dec  3 17:04 diag-9f323b812fd1edc8217715ca0c2ba8e9.svg
-rw-r--r--   1 Lachlan  staff  21137 Dec  3 17:04 diag-e2488cc783b3434076c5a44ee7f19a2b.png
-rw-r--r--   1 Lachlan  staff   7188 Dec  3 17:04 diag-e2488cc783b3434076c5a44ee7f19a2b.svg
-rw-r--r--   1 Lachlan  staff   1508 Dec  3 17:02 index.adoc
...
```

**Comment by sbordet** [comment]
@lachlan-roberts I don't see this.

Please delete these files are try again, perhaps they were generated a while back but not anymore?

**Comment by lachlan-roberts** [comment]
On my Mac I deleted them a few times and they kept coming back, I haven't seen it come back on Ubuntu machine yet, so if I don't see this for a while I will close the issue.

**Comment by janbartel** [comment]
@lachlan-roberts and @sbordet I get these .png files too on Ubuntu when I open the documentation in Eclipse with the asciidoc plugin.   I don't know if it is caused by the plugin, or by some native rendering code that the plugin uses. I haven't found a way of configuring it yet.

**Comment by sbordet** [comment]
Closed by #6509.
